### ISSUE TITLE
cli tool fails user-info.php

### ISSUE THREAD

**Issue by oupala** [FreshRSS/FreshRSS#2291]
I'm trying to install FreshRSS 1.13.1 and I'm using cli tools to check if a user exists before trying to create it:

```bash
>php cli/user-info.php -h --user myuser
*	myuser	2019-03-22T22:10:11+01:00	5.77 MiB	4 categories	20 feeds	909 reads	1 unreads	2 favourites
PHP Fatal error:  Call to a member function execute() on boolean in /path/to/FreshRSS-1.13.1/app/Models/TagDAO.php on line 190
>echo $?
255
```

It seems that the user is found, but there is also an error that is outputted and the return code is 255 instead of 0.

The script output would be more easily readable if there was headers outputted. It could be an option to output the headers or not (`--headers true` or `--headers false`).

**Comment by Alkarex** [comment]
@oupala Thanks for the bug report. Please try https://github.com/FreshRSS/FreshRSS/pull/2292
I am not sure what you mean regarding the headers

**Comment by oupala** [comment]
I've seen that #2292 only adds a `done()`. Have you seen that I'm reporting a **php fatal error** also ? This error explains the error return code.

And for the headers, I was thinking about something like that:

```bash
>php cli/user-info.php -h --user myuser --headers false
*	myuser	2019-03-22T22:10:11+01:00	5.77 MiB	4 categories	20 feeds	909 reads	1 unreads	2 favourites

>php cli/user-info.php -h --user myuser --headers true
admin username last update               total space used # of categories # of feeds # of reads posts # of unreads posts # of favorite posts
-----|--------|-------------------------|----------------|---------------|----------|----------------|------------------|-------------------
*     myuser   2019-03-22T22:10:11+01:00 5.77 MiB         4 categories    20 feeds   909 reads        1 unreads          2 favourites
```

**Comment by Alkarex** [comment]
Ah, sorry, I had not seen that part :-P Please try https://github.com/FreshRSS/FreshRSS/pull/2292 again

Ok for the headers, understood. Edit: Please open a new issue for that

**Comment by oupala** [comment]
Your fix seems to work.

The problem was when a user was not using tags, right?

**Comment by Alkarex** [comment]
Probably a user that has not been active since we introduced tags, independently of whether the user used tags or not.

**Comment by oupala** [comment]
The user was still using FreshRSS v1.11.2 and was migrating to v1.13.1. So I believe that tags were not available in v1.11.2.

**Comment by Alkarex** [comment]
Labels were introduced in FreshRSS 1.12.0.
